feature-libs/checkout/base/components/checkout-billing-address/checkout-billing-address-form.component.ts

```diff
@@ -0,0 +1,200 @@
+        const regionControl = this.billingAddressForm.get(
+          'region.isocodeShort'
+        );
+        if (regions.length > 0) {
+          regionControl?.enable();
+        } else {
+          regionControl?.disable();
+        }
+      })
+    );
+  }
+  billingAddressForm: UntypedFormGroup =
+    this.billingAddressFormService.getBillingAddressForm();
+
+  toggleSameAsDeliveryAddress(): void {
+    this.sameAsDeliveryAddress = !this.sameAsDeliveryAddress;
+    if (this.sameAsDeliveryAddress) {
+      this.deliveryAddress$.subscribe((address) => {
+        this.billingAddressFormService.setDeliveryAddressAsBillingAddress(
+          address
+        );
+      });
+    } else {
+      this.billingAddressFormService.setDeliveryAddressAsBillingAddress(
+        undefined
+      );
+    }
+  }
+  /**
+   *TODO: This method is not used, but should be. It triggers suggested addresses modal under the hood.
+   *
+   * See ticket CXSPA-1276
+   */
+  verifyAddress(): void {
+    if (!this.sameAsDeliveryAddress) {
+      this.userAddressService
+        .verifyAddress(this.billingAddressForm.value)
+        .subscribe((result) => {
+          this.handleAddressVerificationResults(result);
+        });
+    }
+  }
+  protected handleAddressVerificationResults(results: AddressValidation) {
+    if (results.decision === 'ACCEPT') {
+      //continue
+    } else if (results.decision === 'REJECT') {
+      this.globalMessageService.add(
+        { key: 'addressForm.invalidAddress' },
+        GlobalMessageType.MSG_TYPE_ERROR
+      );
+    } else if (results.decision === 'REVIEW') {
+      this.openSuggestedAddress(results);
+    }
+  }
+
+  countrySelected(country: Country): void {
+    this.billingAddressForm.get('country.isocode')?.setValue(country.isocode);
+    this.selectedCountry$.next(country.isocode as string);
+  }
+  //TODO: Add elementRef to trigger button when verifyAddress is used.
+  openSuggestedAddress(results: AddressValidation): void {
+    this.launchDialogService.openDialogAndSubscribe(
+      LAUNCH_CALLER.SUGGESTED_ADDRESSES,
+      undefined,
+      {
+        enteredAddress: this.billingAddressForm.value,
+        suggestedAddresses: results.suggestedAddresses,
+      }
+    );
+    //TODO: Add logic that handle dialog's actions. Scope of CXSPA-1276
+  }
+  getAddressCardContent(address: Address): Observable<Card> {
+    return this.translationService
+      ? combineLatest([
+          this.translationService.translate('addressCard.phoneNumber'),
+          this.translationService.translate('addressCard.mobileNumber'),
+        ]).pipe(
+          map(([textPhone, textMobile]) => {
+            let region = '';
+            if (address.region && address.region.isocode) {
+              region = address.region.isocode + ', ';
+            }
+            const numbers = getAddressNumbers(address, textPhone, textMobile);
+
+            return {
+              textBold: address.firstName + ' ' + address.lastName,
```

Comment:
Small but better to use template literals:
textBold: `${address.firstName} ${address.lastName}`